### ISSUE TITLE
xpra.service configuration (for proxy server) uses incorrect environment file path: Hence startup fails.

### ISSUE THREAD

**Issue by zerofino** [Xpra-org/xpra#3067]
File `/lib/systemd/system/xpra.service` shows:
```
EnvironmentFile=-/etc/sysconfig/xpra
```
But there is no such path. Instead this one is found:
```
/etc/xpra/xpra.conf
```
The last is rather empty and should be used to define the environment variables used later in the service specification file, I guess.
Correction seemed to operate as logs tell, but my system still has more configuration errors before it could run.

**Comment by totaam** [comment]
What platform is this on? What package did you use to install?

On Fedora and RedHat / CentOS, the path you have is the correct one and the file should have been installed there.
On Debian / Ubuntu, it should point to `/etc/default/xpra` instead.

`/etc/xpra/xpra.conf` has nothing to do with it.

**Comment by zerofino** [comment]
Hello Antoine,

my system is a Debian11 (bullseye) started with a Debian9 delivered with Parallels 15, as far as I remember, and upgraded to 10 and 11 via packages.
I hope this helps.
Many thanks for your support!

Best regards,
Harald

**Comment by totaam** [comment]
Got it, eventually: it was hard to figure out because regular builds use the correct path, the problem only occurs when making DEB packages via `debuild`.
That's because the automagic logic in `dh-build` grabs the `xpra.service` file from the source tree and overwrites the one we generate with the correct path in it.

There are `4.2` packages with this fix in the beta repository. This fix will be included in 4.1.3.
Alternatively, to fix a 4.1.2 installation, you can also just run (as root):
```shell
mv /etc/sysconfig/xpra /etc/default/
rmdir /etc/sysconfig
```
